### PR TITLE
Retain store input values

### PR DESCRIPTION
**Pull request by insacuri**
## Checklist
  - [ ] CHANGELOG.md updated? (if required)

### CODE REVIEW

File: console/views/store.py [hunk 1]
```diff
@@ -92,18 +92,31 @@ def reprocess_submission():
...
+        datetime_latest_value = datetime.strptime(datetime_latest, '%Y-%m-%dT%H:%M')
+
     form = StoreForm(
+        valid = valid,
         tx_id=tx_id,
         ru_ref=ru_ref,
-        survey_id=survey_id
+        survey_id=survey_id,
+        datetime_earliest = datetime_earliest_value,
+        datetime_latest = datetime_latest_value
```
Comment (dcdarrell9) [inline console/views/store.py]: These three should have no whitespace... `valid=valid,`

File: console/views/store.py [hunk 2]
```diff
@@ -31,7 +31,7 @@ def get_filtered_responses(logger, valid, tx_id, ru_ref, survey_id, datetime_ear
         if valid == "invalid":
             q = q.filter(SurveyResponse.invalid)
         elif valid == "valid":
-            q = q.filter(not SurveyResponse.invalid)
+            q = q.filter(SurveyResponse.invalid == False)
```
Comment (dcdarrell9) [inline console/views/store.py]: This is fine for sqlalchemy but not python, easiest way would be to add `# noqa`
I think there's a way to import some sqlalchemy stuff to get it compliant, but doesn't seem worth it.

**Review by dcdarrell9 — CHANGES_REQUESTED** [review]
looks good 👍

**Review by dcdarrell9 — APPROVED** [review]

File: console/views/store.py (hunk 1 above)
Comment (insacuri) [reply console/views/store.py]: Old habits die hard with coding standards :'(  Fixed now!

**Pull request merged**